src/authentication/components/AuthCallback.tsx

```diff
@@ -16,7 +17,15 @@ class AuthCallback extends Component<IProps> {
   }
 
   public render() {
-    return <div>Logged in</div>;
+    if (this.props.auth && this.props.auth.user && this.props.auth.user.state) {
```

Comment:
Maybe clean this up a bit?
Like fetching `auth` from `this.props`to avoid writing `this.props` this many times?